book/website/community-handbook/bookdash/bookdash-logistics.md

```diff
@@ -0,0 +1,100 @@
+  * Find volunteers from the organisers who would like to host a GitHub introduction session (1 week before the event)
+  * Find volunteers from the organisers who would like to run social events (discussion session, meal, informal chat) - During the event
+  * Find volunteers from the organisers who would like to run community share out sessions - on the last day
+  * Check with them if there is anything else they would like to suggest or get involved in
+- [ ] Announce in the newsletter and promote on Twitter at least 2-3 months in advance
+- [ ] Think about who else needs to be invited, like Artists, speakers, more diverse participants
+- [ ] Identify helpers from the core members and invite them
+- [ ] Send a reminder in next newsletters with more details if needed
+- [ ] Set an Eventbrite page with a registration code for the selected participants
+- [ ] Close the application round in time (1-2 months ahead of the event)
+- [ ] Set-up a form for reviewing applications (see {ref}`ch-bookdash-application-rubrics` for details)
+- [ ] Review applications in a designated panel (Book Dash Planning Committee) using Book Dash Rubrics for selection
+- [ ] Set up another meeting with the Book Dash Planning Committee to discuss the applications and finalise the selection
+  * Use this meeting to also allocate tasks for during and after the event (indicate that on the GitHub issues)
+- [ ] Send an email to the selected attendees with registration
+- [ ] Send an email to the unselected attendees with feedback
+- [ ] Set up an email chain and a Slack channel to connect all members and share updates
+- [ ] Update presentation for introducing the project to the participants on day-1 of the book dash
+- [ ] Update shared HackMD for `pre-event calls<ch-template-bookdash-precall>`, {ref}`book dash event<ch-template-bookdash-notes>` and {ref}`feedback<ch-template-bookdash-feedback>`
+- [ ] Provide details on Code of Conduct, contribution guideline and ways to get involved in an ongoing discussion
+- [ ] Host the onboarding call one week before the event to share logistics and facilitate the drafting of SMART goals
+- [ ] Group participants into the proposed working groups as per their SMART goals from the onboarding call
+- [ ] Host the onboarding call one week before the event to share logistics and facilitate the drafting of SMART goals
+- [ ] Send a reminder email to register on Eventbrite sharing important links and information including support grant, GitHub session, onboarding call info and reimbursement process
+- [ ] Create a GitHub issue to collect bio and highlight of the participants to add them to the Contributors Record
+
+**Additional task for an online event**
+
+- [ ] State the requirement and expectation of the event
+  * An active GitHub account
+  * Some experience with using GitHub
+  * Familiarity with contribution guidelines and Code of Conduct
+  * A full working day time commitment
```

Comment:
The timings here don't reflect what we have done at online Book Dashes - maybe it should say 'must commit to attend one coworking session a day (2.5 hours) for each day of the Book Dash'.

Reply:
Added this "[ ] Ask them to choose at least one development session (2.5 hours) for each day of the Book Dash to ensure flexibility (rather than blocking the full day)"